lib/core.js

```diff
@@ -1753,6 +1753,16 @@ interface $Iterable<+Yield,+Return,-Next> {
 }
 type Iterable<+T> = $Iterable<T,void,void>;
 
+interface IteratorConstructor {
+    /**
+     * This method allows wrapping "iterator-like" objects with an iterator.
+     * @returns The argument object if it's already an iterator, or a wrapping iterator if the passed object implements a callable @@iterator property.
+     */
+    from<+Yield,+Return,-Next>(source: $Iterable<Yield,Return,Next>): $Iterator<Yield,Return,Next>;
+ }
+
+declare var Iterator: IteratorConstructor;
```

Comment:
This is [how TypeScript declares](9913e9c4496dcc/src/lib/esnext.iterator.d.ts#L129) `Iterator.from`, but Flow does not permit the re-declaration (error: `name-already-bound`). [Try Flow](https://flow.org/try/#1N4Igxg9gdgZglgcxALlAIwIZoKYBsD6uEEAztvhgE6UYCe+JADpdhgCYowa5kA0I2KAFcAtiRQAXSkOz9sADwxgJ+NPTbYuQ3BMnTZA+Y2yU4IwRO4A6SFBIrGVDGM7c+h46fNRLuKxJIGWh8MeT0ZfhYlCStpHzNsFBAMIQkIEQwJODAQfiEyfBE4eWw2fDgofDBMsAALfAA3KjgsXGxxZC4eAw0G-GhcWn9aY3wWZldu-g1mbGqJUoBaCRHEzrcDEgBrbAk62kXhXFxJ923d-cPRHEpTgyEoMDaqZdW7vKgoOfaSKgOKpqmDA+d4gB5fMA-P6LCCMLLQbiLOoYCqgh6-GDYRYIXYLSgkRZkCR4jpddwPfJLZjpOBkO4AX34kA0SRWxgABABhdkAXnZwHpAG4ADpQUUaJ5UbDswFc5D8oW5EANEwkODQJINAAMVgATABWAAsepA9KAA)